api/app/services/gitland/commands/log.rb

```diff
@@ -18,8 +18,9 @@ class Log < GitCommand
       #     end
       #   end
       #
-      def initialize(repository, format: nil, first_parent: false)
+      def initialize(repository, latest_commit_hash: nil, format: nil, first_parent: false)
```

Comment:
Nit: This argument shouldn't be called `latest_commit_hash`.

In the context of this file, this argument is not necessarily a `latest` commit hash. It's just the starting point of the log file. 

The word `latest_commit_hash` only makes sense in the context of the caller, the `RepositorySyncService` class.

Reply:
renamed to commit_hash